lib/tasks/utilities/generate.rb

```diff
@@ -63,7 +63,7 @@ def _get_aws_resource(records, deployment_config)
 def _split_line_gcp(data)
   if data.include? "v=DMARC1" and data.length > 254
```

Comment:
You didn't make this change (I didn't catch it before), but we [generally prefer `&&` to `and`](https://github.com/alphagov/govuk-lint/blob/master/configs/rubocop/gds-ruby-styleguide.yml#L128-L130).